### ISSUE TITLE
One thing fixed, two still broken....

### ISSUE THREAD

**Issue by mistahwolf** [MAFINS/MenyooSP#537]
As of the latest Menyoo update '1.8.1' the issue with changing ped skins and freemode ped clothes crashing the game has been fixed. Thanks for that it was a major pain. Still an issue are marker teleports only working if your avatar is in a vehicle with the 'allow vehicle option' selected. When on foot, for whatever reason markers won't teleport the player by default. Initial investigations pointed to 'No Boundary Limits' being the cause, but the last two 'surprise updates' have rendered NBL a 'non-factor'. Lastly, there's still an issue with loading building props greater than 700 objects. Both have been issues since the 'Contract' update.

As per the issue with objects, what is the exact parameter in the gameconfig that controls the number props menyoo can load in?

**Comment by nomakewan** [comment]
Teleporting to a marker while on foot works perfectly fine for me. To be clear, you mean going into the pause menu, selecting the map, setting a waypoint on the map, then exiting the pause menu, opening Menyoo, going to Teleport Options, and then selecting Waypoint, correct?

Because if so, then this works for me in 1.8.1, and has in fact always worked for me.

As to your question about prop limits: https://forums.gta5-mods.com/post/192813

**Comment by mistahwolf** [comment]
No. 'Markers' spawned with 'Spooner' that allow you to teleport into
interiors have had this issue since the Contract update. Using a marker
from the Spooner menu will allow teleportation into an interior only while
in a vehicle. For whatever reason, passing through a marker on foot does
nothing. The player avatar used to be able to teleport via spawned markers
by default. Now it's like they need an 'allow' option like vehicles do.

**Comment by nomakewan** [comment]
Ah, so you mean mission markers, and aren't referring to using Teleport Options. Interesting; I wonder if this is related to how in a mod I work on, the mission marker only triggers if the player is in a vehicle. I only started modding GTAV after The Contract came out (I started modding February 2022), so I've never experienced earlier builds.

Now I'll have to actually look into this. If it's something I can fix, it'll benefit both Menyoo as well as our own mod. Thank you for the clarification!

**Comment by mistahwolf** [comment]
Thanks. It's stalled completion of several builds. Also still an issue is
after the 'Contract', Menyoo crashes with a scripthook error when trying to
load maps larger than 700 props. I suspect this has something to do with
the 'Map Builder' mod as I've recently loaded one map with '900+' props and
peds with animations and a '780+' mod with 'vanilla props' only just
yesterday with no probs. However, this only started with the 'Contract'
dlc. I've increased both the 'Building and Object Pool Sizes' with no luck.
I've contacted Omegakids (MB's creator) numerous times about this but he's
only interested in helping 'patreons'. (spits) Since the logs all say it's
a 'Menyoo' problem maybe you can look into this. Anyway, thanks for keeping
this most excellent trainer and building mod alive....

**Comment by nomakewan** [comment]
There's no way I can look into that particular bug, I'm afraid. That's not my area of expertise. In fact a lot of C++ stuff isn't really my area of expertise either, but things like syntax problems or altered natives are usually possible to locate and resolve.

But memory management specific to RAGE/SHV and how assets are streamed and things like that are not something I deal with and so without that foundation I couldn't even begin to help you figure out the problem. Reverse engineering is not my wheelhouse.

That said, it is always good practice when reporting a problem to provide the specific error in question rather than saying "a scripthook error." What error was it? What specific text? While I most likely will not be able to help you, being more specific about the error in question may help those who can more quickly identify the issue.

**Comment by mistahwolf** [comment]
The only info given was 'Scripthook Error'. After checking all the logs everything points to Menyoo having an issue with the number of props able to be loaded before crashing the game. I recently updated an old map that had 700+ props and 'hit a wall' upon the '860th' which crashed the game.

Here's the latter part of the shv log where menyoo crashed:
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\needle_orange.png', id 51
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\night_needle_orange.png', id 52
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\needle_red.png', id 53
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\night_needle_red.png', id 54
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_cyan2.png', id 55
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_green2.png', id 56
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_yellow2.png', id 57
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_orange2.png', id 58
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_purple2.png', id 59
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_pink2.png', id 60
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_cyan.png', id 61
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_green.png', id 62
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_yellow.png', id 63
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_orange.png', id 64
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_purple.png', id 65
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Speedo\bg_pink.png', id 66
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Default_face.png', id 67
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Default_handh.png', id 68
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Default_handm.png', id 69
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Frameless_face.png', id 70
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Frameless_handh.png', id 71
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\Clock\Frameless_handm.png', id 72
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\teleportToWaypointBoxKeyboard.png', id 73
[05:23:55] CORE: Creating texture 'D:\Grand Theft Auto V\menyooStuff\Graphics\teleportToWaypointBoxGamepad.png', id 74
[05:23:55] CORE: Launching main() for 'Menyoo.asi' (0x000007FED0469C20), id 39
[05:23:55] CORE: Launching main() for 'OpenInteriors.asi' (0x000007FED997B950), id 40
[05:23:55] CORE: Launching main() for 'Portals.asi' (0x000007FED8804470), id 41
[05:23:55] CORE: Launching main() for 'RANCH_MP_MAPS.asi' (0x000007FED98D1FDC), id 42
[05:24:00] CORE: Launching main() for 'ScriptHookVDotNet.asi' (0x000007FED8651480), id 43
[05:24:00] CORE: Launching main() for 'TrainerV.asi' (0x000007FECF1095D0), id 44
[06:03:25] CORE: An exception occurred while executing 'Menyoo.asi' (0x000007FED045A3A0), id 38

**Comment by mistahwolf** [comment]
BTW, Any luck with what's going on with the 'Mission Markers'? I still can't figure why the teleport function works while in a vehicle, but not on foot?

**Comment by mistahwolf** [comment]
@nomakewan , So... now the markers 'work', but only when I first start the game. Long as I don't get involved with too many peds (cops primarily) or play too long, the markers work perfectly. If the previous two conditions are exceeded, they can only be accessed when in a vehicle.
